apps/crunchy-postgres/workload/test.yml

```diff
@@ -0,0 +1,148 @@
+---
+- hosts: localhost
+  connection: local
+
+  vars_files:
+    - test_vars.yml
+
+  tasks:
+    - block:
+       ## RECORD START-OF-TEST IN LITMUS RESULT CR
+        - name: Generate the litmus result CR to reflect SOT (Start of Test) 
+          template: 
+            src: /litmus-result.j2
+            dest: litmus-result.yaml
+          vars: 
+            test: "{{ test_name }}"
+            app: ""
+            chaostype: ""
+            phase: in-progress
+            verdict: none
+        
+        - name: Apply the litmus result CR
+          shell: kubectl apply -f litmus-result.yaml
+          args:
+            executable: /bin/bash
+          register: lr_status 
+          failed_when: "lr_status.rc != 0"
+
+        - name: Obtaining the loadgen pod label from env.
+          set_fact:
+            loadgen_lkey: "{{ loadgen_label.split('=')[0] }}"
+            loadgen_lvalue: "{{ loadgen_label.split('=')[1] }}"
+
+        - name: Replace the label in loadgen job spec.
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "loadgen_lkey: loadgen_lvalue"
+            replace: "{{ loadgen_lkey }}: {{ loadgen_lvalue }}"
+
+        - name: Replace namespace placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "name_space"
+            replace: "{{ namespace }}"
+
+        - name: Replace service placeholder in Crunchy-loadgen
+          replace:
+            path: "{{ crunchy_loadgen }}"
+            regexp: "servicename"
+            replace: "{{ lookup('env','SERVICE_NAME') }}"
```

Comment:
If we obtain app label, app namespace as variable, this task could be replaced with service detection and its updation to loadgen manifest tasks.

Reply:
Sure i will make the changes